editor/src/core/workers/parser-printer/parse-cache-utils.worker.ts

```diff
@@ -0,0 +1,113 @@
+import { URL_HASH } from '../../../common/env-vars'
+import { type ParseCacheOptions } from '../../../core/shared/parse-cache-utils'
+import type { ParseFile, ParseFileResult } from '../common/worker-types'
+import localforage from 'localforage'
+
+export const CACHE_DB_NAME = 'editor-cache'
+export const PARSE_CACHE_STORE_NAME = 'file-parse-cache'
+const ARBITRARY_CODE_FILE_NAME = 'code.tsx'
```

Comment:
This could be used in `property-controls-local-parser-bridge.ts` to avoid it polluting the cache if the filename changes there